script/deploy-canary-demo

```diff
@@ -9,7 +9,11 @@ cd $GITHUB_WORKSPACE/demos/serverless
 npm ci
 
 echo "Deploying to alpha stage for canary"
-npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -i eu-south-1 -t -l
+npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l
```

Comment:
This and the next is actually are in the same region can you just add an comment like:
```
# Uses Chime Client
npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l

# Uses ChimeMeetingsSDK client
npm run deploy -- -r us-east-1 -b chime-sdk-demo-canary-us-east-1 -s chime-sdk-demo-canary-us-east-1 -o chime-sdk-demo-canary-us-east-1 -t -l
npm run deploy -- -r us-west-2 -b chime-sdk-demo-canary-us-west-2 -s chime-sdk-demo-canary-us-west-2 -o chime-sdk-demo-canary-us-west-2 -t -l
npm run deploy -- -r eu-central-1 -b chime-sdk-demo-canary-eu-central-1 -s chime-sdk-demo-canary-eu-central-1 -o chime-sdk-demo-canary-eu-central-1 -t -l
npm run deploy -- -r ap-southeast-1 -b chime-sdk-demo-canary-ap-southeast-1 -s chime-sdk-demo-canary-ap-southeast-1 -o chime-sdk-demo-canary-ap-southeast-1 -t -l
```

Reply:
fixed